### ISSUE TITLE
Verifable Credential tutorial broke with v0.1.6

### ISSUE THREAD

**Issue by nick-verida** [verida/documentation#96]

**Comment by nick-verida** [comment]
When issuing a VC I get:

```
Uncaught (in promise) TypeError: r is undefined
    p credentials.js:240
    a credentials.js:44
    a credentials.js:25
    i credentials.js:19
    i credentials.js:15
    createCredentialJWT credentials.js:234
    issueVC 5529.14076858.js line 8 > Function:53
    React 11
    unstable_runWithPriority scheduler.production.min.js:18
```

**Comment by nick-verida** [comment]
Fixed by #97
